src/Swashbuckle.AspNetCore.SwaggerUI/Template/index.html

```diff
@@ -1,144 +1,129 @@
+    </symbol>
+
+    <symbol viewBox="0 0 20 20" id="large-arrow-down">
+      <path d="M17.418 6.109c.272-.268.709-.268.979 0s.271.701 0 .969l-7.908 7.83c-.27.268-.707.268-.979 0l-7.908-7.83c-.27-.268-.27-.701 0-.969.271-.268.709-.268.979 0L10 13.25l7.418-7.141z"/>
+    </symbol>
+
+
+    <symbol viewBox="0 0 24 24" id="jump-to">
+      <path d="M19 7v4H5.83l3.58-3.59L8 6l-6 6 6 6 1.41-1.41L5.83 13H21V7z"/>
+    </symbol>
 
-      window.swaggerUi.load();
+    <symbol viewBox="0 0 24 24" id="expand">
+      <path d="M10 18h4v-2h-4v2zM3 6v2h18V6H3zm3 7h12v-2H6v2z"/>
+    </symbol>
 
-      function log() {
-        if ('console' in window) {
-          console.log.apply(console, arguments);
-        }
-      }
+  </defs>
+</svg>
+
+<div id="swagger-ui"></div>
+
+<script src="./swagger-ui-bundle.js"> </script>
+<script src="./swagger-ui-standalone-preset.js"> </script>
+<script>
+window.JSConfig = JSON.parse('%(JSConfig)');
+
+window.onload = function() {
+
+  function log() {
+    if ('console' in window) {
+      console.log.apply(console, arguments);
+    }
+  }
+  
+  // Build a system
+  const ui = SwaggerUIBundle({
+    url: JSConfig.SwaggerEndpoints[0].Url,
+    validatorUrl: JSConfig.ValidatorUrl,
+    dom_id: '#swagger-ui',
+    deepLinking: true,
+    supportedSubmitMethods: JSConfig.SupportedSubmitMethods,
+    docExpansion: JSConfig.DocExpansion,
+    defaultModelRendering: 'schema',
+    showRequestHeaders: JSConfig.ShowRequestHeaders,
+    presets: [
+      SwaggerUIBundle.presets.apis,
+      SwaggerUIStandalonePreset
+    ],
+    plugins: [
+      SwaggerUIBundle.plugins.DownloadUrl
+    ],
+    layout: "StandaloneLayout",
+    onFailure: function(data) {
+      log("Unable to Load SwaggerUI");
+
+      _.each(JSConfig.OnFailureScripts, function (script) {
+        $.getScript(script);
+      });
+    },
+  });
+  
+  ui.initOAuth({
+    clientId: JSConfig.OAuth2ClientId,
+    clientSecret: JSConfig.OAuth2ClientSecret,
+    realm: JSConfig.OAuth2Realm,
+    appName: JSConfig.OAuth2AppName,
+    scopeSeparator: JSConfig.OAuth2ScopeSeparator,
+    additionalQueryStringParams: JSConfig.OAuth2AdditionalQueryStringParams
   });
-  </script>
-</head>
 
-<body class="swagger-section">
-<div id='header'>
-  <div class="swagger-ui-wrap">
-    <a id="logo" href="http://swagger.io"><img class="logo__img" alt="swagger" height="30" width="30" src="images/logo_small.png" /><span class="logo__title">swagger</span></a>
-    <form id='api_selector'>
-      <div class='input'>
-        <input placeholder="http://example.com/api" id="input_baseUrl" name="baseUrl" type="text" />
-        <div id='auth_container'></div>
-        <select id="select_document"></select>
-        <script type="text/javascript">
-          // Append configured endpoints as options in the document selector
```

Comment:
You removed the "select document" - dropdown. 
Know you can add multiple swagger endpoint but you cant switch between them on the UI.